t/APISIX.pm

```diff
@@ -81,7 +81,11 @@ apisix:
   stream_proxy:
     tcp:
       - 9100
-  admin_key: null
+  admin_key:
+    -
+      name: "admin"
+      key: YOUR_API_KEY
```

Comment:
`YOUR_API_KEY` can not be the key

Reply:
it only for testing. maybe `TEST_API_KEY` is a better value